docs/Tutorials/Advanced-Tutorials/Daily-Quests/Delays.md

```diff
@@ -0,0 +1,116 @@
+---
+icon: material/folder-play-outline
+tags:
+  - Delay
+  - Daily Quest
+---
+
+The next tutorial focuses on creating a delay that resets our daily quest after 24 hours. This will ensure players can 
+complete another quest after a full day has passed. Follow along to implement this feature seamlessly.
+
+<div class="grid" markdown>
+!!! example "Related Docs"
+    * [Events List](../../../Documentation/Scripting/Building-Blocks/Events-List.md)
+    * [Objectives List](../../../Documentation/Scripting/Building-Blocks/Objectives-List.md)
+</div>
+
+## 1. Start a delay event
+
+Open the `events.yml` from your QuestPackage and add the following content to it
+
+``` YAML title="events.yml" linenums="1"
+events: # (1)!
+  start_reset_timer: "objective add reset_timer"
+```
+
+1. All events must be defined in an `events` section.
+
+Afterwards we add a new objective into our `objectives.yml` where we start the delay.
+
+``` YAML title="events.yml" hl_lines="11" linenums="1"
+events:
+  pickdaily: "pickrandom 50%daily1,50%daily2"
+  
+  daily1: 'objective add mine_diamonds'
+  daily2: 'objective add mine_iron'
+  
+  dailyComplete: 'folder addQuestDoneTag,NotifyPlayer'
+  notifyPlayer: 'notify You mined enough!\nReturn to the Blacksmith! io:Title sound:firework_rocket'
+  addQuestDoneTag: 'tag add questDone'
+  
+  start_reset_timer: "objective add reset_timer"
+```
+
+Before we move on to our `objectives.yml`, let's add a folder event and several smaller events to remove all tags and
+objectives that the player might have. In this example, we assume the player has either a `startedQuest` or a `questDone` 
+tag. We will remove both and all possible objectives in case a player did not finish the quest within 24 hours.
+
+
+``` YAML title="events.yml" hl_lines="5 7 12 14-15 19" linenums="1"
+events:
+  pickdaily: "pickrandom 50%daily1,50%daily2"
+  
+  daily1: 'objective add mine_diamonds'
+  daily1_r: 'objective remove mine_diamonds'
+  daily2: 'objective add mine_iron'
+  daily2_r: 'objective remove mine_iron'
+  
+  dailyComplete: 'folder addQuestDoneTag,NotifyPlayer'
+  notifyPlayer: 'notify You mined enough!\nReturn to the Blacksmith! io:Title sound:firework_rocket'
+  addQuestDoneTag: 'tag add questDone'
+  addQuestDoneTag_r: 'tag add questDone'
+  
+  addQuestStartedTag: 'tag remove questStarted' # (1)!
+  addQuestStartedTag_r: 'tag remove questStarted'
+  
+  start_reset_timer: 'objective add reset_timer'
+  
+  folderReset: 'folder daily1_r,daily2_r,addQuestStartedTag_r,addQuestDoneTag_r'
```

Comment:
As I wrote above I would use schedules :D But this can still be used for the schedules!
And here are some mistakes I guess that needs to be looked at
```suggestion
events:
  pickdaily: "pickrandom 50%daily_1,50%daily_2"
  
  addDaily_1: 'objective add mineDiamonds'
  removeDaily_1: 'objective remove mineDiamonds'
  addDaily_2: 'objective add mineIron'
  removeDaily_2: 'objective remove mineIron'
  
  dailyComplete: 'folder addQuestDoneTag,notifyPlayer'
  notifyPlayer: 'notify You mined enough!\nReturn to the Blacksmith! io:Title sound:firework_rocket'
  addQuestDoneTag: 'tag add questDone'
  removeQuestDoneTag: 'tag remove questDone'
  
  addQuestStartedTag: 'tag add questStarted' # (1)!
  removeQuestStartedTag: 'tag remove questStarted'
  
  startResetTimer: 'objective add resetTimer'
  
  folderReset: 'folder removeDaily_1,removeDaily_2,removeQuestStartedTag,removeQuestDoneTag'
```